src/betterbib/tools.py

```diff
@@ -234,7 +234,15 @@ def pybtex_to_bibtex_string(
 
         try:
             if key not in ["url", "doi"]:
+                # Parse the original value to get a unified version
+                value = codecs.decode(value, "ulatex")
                 value = codecs.encode(value, "ulatex")
+
+                # Replace doubled spaces
+                value = value.replace("  ", " ")
```

Comment:
Shouldn't we better regex-replace multiple spaces here?

Reply:
I  don't think so, as `latexcodec` already "eats" multiple spaces except for commands, where it inserts them deliberatly.